README.md

```diff
@@ -1,6 +1,8 @@
 # Local-links-manager
 
-Admin interface for managing Local Authorities links including all their services and interactions.
+Admin interface for managing Local Authorities' links including all their services and interactions.
+
+For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL on a Local Authority's website that contains information on disposing of garden waste.
```

Comment:
I think there may be a typo below:

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to provide the URL of a Local Authority's website that contains information on disposing of garden waste.
```

You can click "Apply suggestion" (or whatever it is called) to have GitHub automatically create an additional commit for you. Though we typically prefer rebasing our commits, which may be a Git step too far at the moment!

Reply:
Thanks @ChrisBAshton. I did actually mean 'on', because the Local Links Manager should give you the specific URL, not the homepage (which is what I would understand from 'of'). But I agree it's not quite clear at the moment. How about:

`For example, this service uses Local Links Manager to provide the specific URL, on each Local Authority's website, with information about how to dispose of garden waste.`

Reply:
Ah, I see what you mean. And I like your new proposed text, but think it could do with a final tweak to make it clear it's talking about one Local Authority (not multiple):

```suggestion
For example, [this service](https://www.gov.uk/garden-waste-disposal) uses Local Links Manager to determine your Local Authority and links to the garden waste disposal page on the Local Authority's website.
```

Reply:
Thanks @ChrisBAshton. Local Links Manager doesn't actually determine a user's Local Authority - that's what Mapit does. I've removed that from your text, but have added a second sentence to make clear the difference between the two tools. (I'm not mentioning Mapit - our current postcode lookup tool - by name, because we're planning to replace it.)

```
For example, this service uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.

Note that Local Links Manager does not determine a user's Local Authority based on their postcode. We use a separate postcode lookup tool for that.
```

Reply:
Looks good 👍 I think it's worth replacing `We use a separate postcode lookup tool for that.` with `We use [Mapit][LINK] for that.` But otherwise I think we're good to go, and have had plenty of practice in the GitHub review process! :)

Reply:
Thanks @ChrisBAshton! Final suggested text:

```
For example, this service uses Local Links Manager to determine the URL of the garden waste disposal page on the Local Authority website which corresponds to the postcode a user has entered.

Note that Local Links Manager does not determine a user's Local Authority based on their postcode. We use [Mapit](https://github.com/alphagov/mapit) for that.

``` 

- Now that I've linked to Mapit in the code, and we know that we're going to replace it, is there a standard place I can make a note of this, so that we know to update this text when we get rid of Mapit?
- I'm not sure how to make another change to my code (but would like to learn). I think a quick pair would be the best for this. I'll ask for another pair on #govuk-platform-health Slack.